### ISSUE TITLE
Propose adding WCAG validation to build process

### ISSUE THREAD

**Issue by jrit** [nodejs/nodejs.org#552]
Now that most of the colors are updated to better contrast in #536 I think it would be a good idea to add WCAG validation into the build process to help clean up anything else that is not compliant, and to prevent regression in future design changes. I'd be willing to take this on, but wanted to get a consensus that it is desired first.

I am assuming that validation will point to some minor issues, so there will likely be some work to resolve those before the validation itself could be merged since otherwise it would fail builds.

**Comment by silverwind** [comment]
Would that require something like PhantomJS as a test dependency?

**Comment by jrit** [comment]
Yes, if we use https://www.npmjs.com/package/accessibility-developer-tools which is what I'd recommend, then it runs within a browser, and PhantomJS is generally the easiest to configure in CI. Would that create a problem @silverwind ?

**Comment by silverwind** [comment]
To be honest, I'm not sure it's worth the complexity it adds for something that can be done manually once in a while.

**Comment by jrit** [comment]
I'll spend some time with it and see if there is an effective but less burdensome way to set it up. Possibly just a simple to run script that is not part of the build process. Thanks for your thoughts.

**Comment by jrit** [comment]
I'm closing since manual checking seems pretty reasonable to me after working on some of the validation. A weekly review of PRs for issues is probably a good idea, I'll try and stay on top of it.
